ce/api/util.py

```diff
@@ -95,14 +96,30 @@ def mean_datetime(datetimes):
     mean = np.mean(timestamps)
     return datetime.fromtimestamp(mean, tz=timezone.utc)
 
+def validate_cell_method(cell_method):
+    return cell_method in ('mean', 'standard_deviation')
+
 def search_for_unique_ids(sesh, ensemble_name='ce', model='', emission='',
-                          variable='', time=0, timescale=''):
+                          variable='', time=0, timescale='', cell_method='mean'):
+    if not validate_cell_method(cell_method):
+        raise Exception('Unsupported cell_method: {}'.format(cell_method))
+
+    cell_methods = sesh.query(mm.DataFileVariable.variable_cell_methods)\
+                    .distinct(mm.DataFileVariable.variable_cell_methods).all()
+    pattern = {
+        'standard_deviation': r'time: [a-z]* time: standard_deviation over days',
+        'mean': r'time: [a-z]*( time: mean over days)?'
```

Comment:
These regexes aren't robust to whitespace. I don't think there is anything in the CF standard that says the space between words must be exactly one space. Suggest you make it more robust by allowing `\s+` between words.

Reply:
Testing robustness would be nice but is probably not necessary.